assets/js/blocks/cart-checkout/checkout/block.js

```diff
@@ -324,6 +329,18 @@ const Block = ( {
 								}
 							/>
 						</FormStep>
+						<div className="wc-block-checkout__actions">
+							{ attributes.showReturnToCart && (
+								<ReturnToCartButton
+									link={
+										getSetting(
+											'page-' + attributes.cartPageId
+										) || CART_URL
```

Comment:
`getSetting` can have a default, maybe move `CART_URL` in as the default here? Alternatively, the `ReturnToCartButton` _already_ uses `CART_URL` if the link prop is undefined or an empty string... so maybe just pass in the result of the `getSetting` call?